server/app/services/mail-service.js

```diff
@@ -190,11 +192,118 @@ async function sendWelcomeMail(email) {
   await transporter.sendMail(mailOptions);
 }
 
+/**
+ * Send a contact form mail
+ *
+ * @param formName {String} french contact form name
+ * @param emails {String[]} recipient email address
+ * @param value {Object} form value
+ * @returns {Promise<void>} Nothing
+ */
+async function sendContactForm(formName, emails, values) {
+  const mail = await readFile(path.resolve(__dirname, '../../resources/emails', 'contact.html'), 'utf8');
+
+  // eslint-disable-next-line
+  for (const email of emails) {
+    const currentMail = mail.replace(REGEX_TOKEN, (matches, replaceToken) => {
+      switch (replaceToken) {
+        case 'MAIL_WEBSITE':
+          return WEB_CONFIG.publicURL;
+        case 'MAIL_USERNAME':
+          return email;
+        case 'FORM_NAME':
+          return formName;
+        case 'FORM_VALUES': {
+          let html = '<p><ul>';
+          Object.keys(values).forEach((value) => {
+            html += `<li><b>${TRANSLATION[value] ? TRANSLATION[value].french : value}</b>: ${values[value]}</li>`;
+          });
+          html += '</ul></p>';
+          return html;
+        }
+        default:
+          return null;
+      }
+    });
+
+    const transporter = nodemailer.createTransport(CONFIG);
+    // Setup email data with unicode symbols
+    const mailOptions = {
+      from: CONFIG.auth.user,
+      to: email,
+      subject: `[K-App] Nouveau formulaire de contact pour un ${formName}`,
+      html: currentMail,
+    };
+
+    // eslint-disable-next-line
+    await transporter.sendMail(mailOptions);
+  }
+}
+
+/**
+ * Create mail template from ressource
+ *
+ * @param emailBody {String} mail body from ressources/email/body.js
+ * @returns {String} mail template
+ */
+function createMailTempalte(emailBody) {
+  return EMAIL_TEMPLATES.HEADER + emailBody + EMAIL_TEMPLATES.FOOTER;
+}
+
+async function sendMail(to, subject, html) {
+  const transporter = nodemailer.createTransport(CONFIG);
+
+  const mailOptions = {
+    from: CONFIG.auth.user,
+    to,
+    subject,
+    html,
+  };
+
+  await transporter.sendMail(mailOptions);
+}
+
+async function sendContactFormV2(formName, emails, values) {
+  const emailTemplate = createMailTempalte(EMAIL_TEMPLATES.CONTACT)
+    .replace(REGEX_TOKEN, (matches, replaceToken) => {
+      switch (replaceToken) {
+        case 'FORM_NAME':
+          return formName;
+        case 'FORM_VALUES': {
+          let html = '<p><ul>';
+          Object.keys(values).forEach((value) => {
+            html += `<li><b>${TRANSLATION[value] ? TRANSLATION[value].french : value}</b>: ${values[value]}</li>`;
```

Comment:
Do we really need translation? We don't use it anywhere else

Reply:
We don't need it but the JSON value received are in english. I was thinking that transmit the value in english and translate it in the backend should be a better way.